sigopt/interface.py

```diff
@@ -97,6 +99,25 @@ def __init__(self, requestor, api_url=None):
       ],
     )
 
+    checkpoints = ApiResource(
```

Comment:
Right now this corresponds to the HTTP endpoint of `/experiments/X/checkpoints`, but it sounds like you want this to match `/experiments/X/training_run/Y/checkpoints`. To do that, you'll want checkpoints to be a sub objective `training_run`